app/services/fact_importer.rb

```diff
@@ -15,8 +15,9 @@ def self.authorized_smart_proxy_features
   end
 
   def self.excluded_facts_regex
+    excluded_facts = ['foreman_uuid_signature', 'smart_proxy_uuid_signature'] + Setting[:excluded_facts]
```

Comment:
Should we ensure that `foreman_uuid` and `smart_proxy_uuid` are never filtered? Otherwise users might (accidentally) drop that fact, making the "only allow special users if uuid fact is set" logic noop?

Reply:
The facts come in, we create the facets, then the facts get filtered and the reduced set of facts gets stored. Even if the users filter out the uuid facts, it should not affect the whole flow